hips/draw/simple.py

```diff
@@ -30,13 +32,21 @@ def draw_sky_image(geometry: WCSGeometry, tiles: Generator[HipsTile, Any, Any],
         A list of HipsTile
     hips_survey : `~hips.HipsSurveyProperties`
         HiPS survey properties
+    tile_format : `str`
```

Comment:
Why pass this around separately?
It's a property of https://hips.readthedocs.io/en/latest/api/hips.HipsSurveyProperties.html#hips.HipsSurveyProperties.tile_format , no?